### PR TITLE
docs: created a new documentation for EKS

### PR DESCRIPTION
**Pull request by balaharish7**

### CODE REVIEW

File: docs/main/Solutioning/openebs-on-kubernetes-platforms/EKS/eks.md [hunk 1]
```diff
@@ -0,0 +1,547 @@
...
+keywords:
+ - Replicated PV Mayastor Installation on Amazon Elastic Kubernetes Service
+ - Replicated PV Mayastor Installation on EKS
+ - Replicated PV Mayastor - Platform Support
+ - Platform Support
+ - Elastic Kubernetes Service
+ - EKS
+description: This section explains about the Platform Support for Replicated PV Mayastor.
+---
+# Replicated PV Mayastor Installation on Amazon Elastic Kubernetes Service
```
Comment (avishnu) [inline docs/main/Solutioning/openebs-on-kubernetes-platforms/EKS/eks.md:13]: What is the main difference between this and the above document?

Comment (balaharish7) [reply docs/main/Solutioning/openebs-on-kubernetes-platforms/EKS/eks.md:13]: Both are for EKS! One for instance store like Local SSD in GKE & the other one is without Local SSD meaning disks can be attached manually to those instances. @avishnu

File: docs/main/Solutioning/openebs-on-kubernetes-platforms/EKS/eks-isv.md [hunk 2]
```diff
@@ -0,0 +1,526 @@
...
+
+This document provides instructions for installing Replicated PV Mayastor on Amazon Elastic Kubernetes Service (EKS) with Instance Store Volumes. Replicated PV Mayastor is designed to work with Amazon EKS with Instance Store Volumes. It provides high-performance storage for stateful applications running in a Kubernetes environment. 
+
+Using OpenEBS Replicated PV Mayastor in EKS with Instance store volumes addresses many of the limitations associated with the ephemeral nature of local SSDs by introducing a layer of persistent storage management. Here's how OpenEBS helps mitigate these limitations:
+
+**Data Persistence through Replication**
+
+- OpenEBS abstracts the underlying storage (Instance store volumes) into a set of persistent volumes. Even if local SSDs are inherently ephemeral, OpenEBS can ensure data persistence by replicating the data across multiple worker nodes.
+
+- For instance, using OpenEBS with replication (e.g., 2 or 3 replicas), ensures that even if one node fails or is terminated, the data exists on other nodes, avoiding data loss.
```
Comment (avishnu) [inline docs/main/Solutioning/openebs-on-kubernetes-platforms/EKS/eks-isv.md:23]: Is it e.g. or eg. ?

File: docs/main/Solutioning/openebs-on-kubernetes-platforms/EKS/eks.md [hunk 3]
```diff
@@ -0,0 +1,547 @@
...
+
+- **Helm Install Command**
+
+```
+helm install openebs --namespace openebs openebs/openebs --create-namespace
+```
+
+## Pools
+
+The available local SSD disks on worker nodes can be viewed by using the `kubectl-mayastor` plugin.
```
Comment (avishnu) [inline docs/main/Solutioning/openebs-on-kubernetes-platforms/EKS/eks.md:78]: Is this statement correct? I think this was for instance volumes.

File: docs/main/Solutioning/openebs-on-kubernetes-platforms/EKS/eks-isv.md (hunk 2 above)
Comment (balaharish7) [reply docs/main/Solutioning/openebs-on-kubernetes-platforms/EKS/eks-isv.md:23]: It's "e.g." @avishnu . Or we can write it as "Example"

File: docs/main/Solutioning/openebs-on-kubernetes-platforms/EKS/eks.md (hunk 3 above)
Comment (balaharish7) [reply docs/main/Solutioning/openebs-on-kubernetes-platforms/EKS/eks.md:78]: It's possible to view for both instances store/local ssd volumes and normal instances.

**Review by avishnu — APPROVED** [review]

**Pull request merged**